Content.Server/Chat/Systems/ChatSystem.cs

```diff
@@ -614,7 +614,8 @@ private void SendDeadChat(EntityUid source, ICommonSession player, string messag
 
     #region Utility
 
-    private enum MessageRangeCheckResult {
+    private enum MessageRangeCheckResult
+    {
```

Comment:
There's some instances of my linter cleaning things up - I've seen guidance on PRs here that fixes should be minimal here, so let me know if I need to clonk my linter on the head.

Reply:
Eh, we do allow some level of formatting (and warnings cleanup for that matter) IF it follows our usual guidelines/style AND there are changes other than formatting AND the formatting changes aren't 90% of the PR, so this is very much allowed

Reply:
Still should prob vibe check your formatter just so the third case I described doesn't happens in the future